monitoring/api/v3/alerts-client/snippets.py

```diff
@@ -0,0 +1,307 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import argparse
+import json
+import os
+
+from google.cloud import monitoring_v3
+import google.protobuf.json_format
+import tabulate
+
+
+def list_alert_policies(project_name):
+    client = monitoring_v3.AlertPolicyServiceClient()
+    policies = client.list_alert_policies(project_name)
+    print(tabulate.tabulate(
+        [(policy.name, policy.display_name) for policy in policies],
+        ('name', 'display_name')))
+
+
+# [START monitoring_alert_list_channels]
+def list_notification_channels(project_name):
+    client = monitoring_v3.NotificationChannelServiceClient()
+    channels = client.list_notification_channels(project_name)
+    print(tabulate.tabulate(
+        [(channel.name, channel.display_name) for channel in channels],
+        ('name', 'display_name')))
+# [END monitoring_alert_list_channels]
+
+
+# [START monitoring_alert_enable_policies]
+def enable_alert_policies(project_name, enable, filter_=None):
+    """Enable or disable alert policies in a project.
+
+    Arguments:
+        project_name {str}
```

Comment:
Not quite Google-style docstrings here. https://sphinxcontrib-napoleon.readthedocs.io/en/latest/#google-vs-numpy

- parentheses instead of squiggly brackets
- colon instead of dashes.